README.md

```diff
@@ -0,0 +1,53 @@
+# chromez
+
+Dashboard for displaying status for Chrome developers.
+
+## Setup
+
+Install [Google App Engine SDK for Python](https://cloud.google.com/appengine/downloads#Google_App_Engine_SDK_for_Python)
+
+Install [npm and nodejs](https://nodejs.org/en/)
+```shell
+wget https://nodejs.org/dist/v4.4.4/node-v4.4.4-linux-x64.tar.xz
+tar -xvf node-v4.4.4-linux-x64.tar.xz
+export PATH=$PATH:$PWD/node-v4.4.4-linux-x64/bin
+```
+
+Install [Bower](http://bower.io/#install-bower)
+```shell
+npm install -g bower
+```
+
+Install [Polymer](https://www.polymer-project.org/1.0/docs/start/getting-the-code.html)
+```shell
+bower install --save Polymer/polymer
```

Comment:
I think you just want to install the bower dependencies with `bower install`

`bower install --save Polymer/polymer` will install the latest version of polymer and then update the `bower.json` file to reference that (but any other dependencies would not be installed).